pkgs/applications/system/asusctl/default.nix

```
@@ -0,0 +1,102 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, e2fsprogs
+, fontconfig
+, libGL
+, libglvnd
+, pkg-config
+, rustPlatform
+, udev
+, xorg
+, xwayland
+, systemd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "asusctl";
+  version = "4.5.0";
+  nativeBuildInputs = [
+    pkg-config
+    cmake
+  ];
+
+  buildInputs = [
+    udev
+    fontconfig
+    xwayland
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXrandr
+    xorg.libXi
+    e2fsprogs
+    libGL
+    libglvnd
+    systemd
+  ];
+
+  src = fetchFromGitLab {
+    owner = "asus-linux";
+    repo = "asusctl";
+    rev = version;
+    sha256 = "sha256-hkNVgMugTllDqnanVU1rdb7viRG4U1f588rLA55yCbY=";
+  };
+
+  cargoSha256 = "sha256-0hPY1+ZlJZfqnWZ+GX1IkhsjU0J0CjrRkegA1DD2yC4=";
+
+  postPatch = ''
+    sed -i -e s,/usr/share,$out/share, daemon/src/config.rs
+    sed -i -e s,/usr/share,$out/share, daemon-user/src/daemon.rs
+    sed -i -e s,/usr/share,$out/share, daemon-user/src/ctrl_anime.rs
+    sed -i -e s,/usr/share,$out/share, daemon-user/src/user_config.rs
+    sed -i -e s,/usr/share,$out/share, rog-control-center/src/main.rs
```

```suggestion
    for file in daemon/src/config.rs daemon-user/src/daemon.rs daemon-user/src/ctrl_anime.rs daemon-user/src/user_config.rs rog-control-center/src/main.rs; do
      sed -i -e s,/usr/share,$out/share, $file
    done
```

or can sed do multiple files natively? Not sure if this is better or not.

---

Done